substrate/client/executor/polkavm/src/lib.rs

```diff
@@ -26,14 +26,15 @@ use sp_wasm_interface::{
 };
 
 #[repr(transparent)]
-pub struct InstancePre(polkavm::InstancePre<()>);
+pub struct InstancePre(polkavm::InstancePre<(), polkavm::CallError>);
 
 #[repr(transparent)]
-pub struct Instance(polkavm::Instance<()>);
+pub struct Instance(polkavm::Instance<(), polkavm::CallError>);
 
 impl WasmModule for InstancePre {
 	fn new_instance(&self) -> Result<Box<dyn WasmInstance>, Error> {
-		Ok(Box::new(Instance(self.0.instantiate()?)))
+		// FIXME: Does not return `sc_executor_common::error::Error`.
+		Ok(Box::new(Instance(self.0.instantiate().unwrap())))
```

Comment:
There shouldn't be an `unwrap` here.

Reply:
https://github.com/paritytech/polkadot-sdk/pull/6533/commits/cfa00ce2a01fd2cedb086c6db4c973d549aa3e76